src/integrationtest/java/com.microsoft.aad.msal4j/DeviceCodeIT.java

```diff
@@ -57,8 +57,8 @@ public void DeviceCodeFlowADTest(String environment) throws Exception {
         Assert.assertFalse(Strings.isNullOrEmpty(result.accessToken()));
     }
 
-    @Test(dataProvider = "environments", dataProviderClass = EnvironmentsProvider.class)
-    public void DeviceCodeFlowADFSv2019Test(String environment) throws Exception {
+    @Test()
```

Comment:
Why are you getting rid of environments for some of the tests?

Reply:
It didn't seem like environment was actually being used in those those tests. All of the config (like user or authority) referenced TestConstants directly, so defining the environments just made the exact same test run twice.